CONTRIBUTING.md

```diff
@@ -359,6 +369,7 @@ mage testSmoke
 
 Smoke tests test Porter using the CLI and quickly identify big problems with a
 build that would make it unusable.
+We recommend always running the smoke tests locally before submitting a pull request that changed Porter's code (not docs), to validate that core functionality wasn't broken.
```

Comment:
I had to re-read this a few times to identify what it's saying. Here's a way that might make it clearer:

```suggestion
If you intend to change Porter code (not docs), we recommend running smoke tests locally before submitting a pull request with your changes. Doing so allows you to immediately ensure you haven't broken or altered any core functionality.
```